First/dailyExercises/checkArraysEqual.java

```diff
@@ -0,0 +1,37 @@
+package firstPackage.First.dailyExercises;
+
+public class checkArraysEqual {
```

Comment:
Class name should start with Capital letter

Reply:
Done !